concrete/src/Tree/Node/Type/Category.php

```diff
@@ -40,7 +41,9 @@ public function getTreeNodeTypeName()
 
     public function getTreeNodeDisplayName($format = 'html')
     {
-        if ($this->getTreeNodeName()) {
+        $app = Facade::getFacadeApplication();
+        $vsh = $app->make('helper/validation/strings');
+        if ($vsh->notempty($this->getTreeNodeName())) {
```

Comment:
Or simply:
```php
if ((string) $this->getTreeNodeName() !== '') {
```

Reply:
We should standardize these things. 
Yeah, I know using the simpler way is faster. But at the same time as the helper method exists in the core I guess it's better to use that one. So that we can use the same way in everywhere. 
I found different types of usage in core. So wasn't sure which one would be the best.